### PR TITLE
Fixes #636 - Move commandline arguments to toml config file

### LINKED ISSUE #636: Move commandline arguments to toml config file
Currently JSON output formatting is defined using the following parameters:

```
  -o, --output=<FILENAME>   Path/filename of the output file. Path can be followed
                            by events the user wishes to output example:
                            -o=artemis.json Eth2Genesis TimeSeriesRecord
  -p, --provider=<PROVIDER TYPE>
                            Output provider types: CSV, JSON (default: JSON).
  -s, --sim                 PoW simulation flag, w/ optional input file
```

Those parameters are the passed to `BeaconNode.class` which instantiates a `FileProvider.class` and an `EventHandler.class` to subscribe and output events specified in the `-o` or `--output`. ie: `-o=artemis.json Eth2Genesis TimeSeriesRecord`.

Those parameters should be removed as configurable commandline arguments and added as arguments to the `toml` file. Doing so should maintain the same current application behavior.

### PR DESCRIPTION
**Pull request by mankenavenkatesh**
## PR Description
1. Removed Arguments From CommandLineArguments.java.
2. Added them to config.toml.
3. Updated ArtemisConfiguration.java with Schema and Getters.
4. Updated BeaconNode.java, PowchainService.java with logic change.

## Fixed Issue(s)

resolves #636

### CODE REVIEW

**Comment by mankenavenkatesh** [comment]
Moved these arguments - simulation, providerType, outputFile from command line to config. 

Output events that are specified after outputfile are not moved to Config yet.
@jrhea @dangerousfood  Please let me know if output events also has to be moved to Config as part of #636.

File: util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java [hunk 1]
```diff
@@ -48,13 +37,6 @@
...
 
-  @Option(
-      names = {"-s", "--sim"},
-      arity = "0",
-      paramLabel = "<FILENAME>",
-      description = "PoW simulation flag, w/ optional input file")
-  private String inputFile = null;
-
   @Option(
       names = {"-f", "--format"},
```
Comment (jrhea) [inline util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java]: go ahead and remove this flag too and put it in the toml config under output.

File: util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java [hunk 2]
```diff
@@ -73,18 +55,6 @@
         }
       };
```
Comment (jrhea) [inline util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java]: Move these Events to the toml file under output as well.

**Review by jrhea — CHANGES_REQUESTED** [review]
@mankenavenkatesh awesome work! seriously, thanks for contributing.  Do you mind implementing the two suggestions i made?  Then i will approve.  Thanks!

File: util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java (hunk 1 above)
Comment (mankenavenkatesh) [reply util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java]: Now i have moved "format" flag to config. Pls review.

File: util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java (hunk 2 above)
Comment (mankenavenkatesh) [reply util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java]: Now i have moved events to config under output. pls review.

**Review by dangerousfood — APPROVED** [review]
Bravo. Thank you Venkatesh

**Review by jrhea — APPROVED** [review]
lgtm

**Pull request merged**